VALUES.md

```diff
@@ -0,0 +1,30 @@
+# Ayo.js Community Values
+
+We don't care how you pronounce it: be it "ey-yo", "I-oh", or "Awooooo" -- the Ayo.js project strives to create an environment where you can feel at home and focus on your contributions to this nice shared space that we all benefit from.
```

Comment:
I agree that it is a slightly odd start. Honestly, swapping this paragraph and the next ("Ayo.js is about the humans that help make it happen...") would solve this, in my opinion.

Alternatively the start of the paragraph could be changed to something like "You can pronounce it however you want:"